sysfs/class_fibrechannel.go

```diff
@@ -153,6 +153,9 @@ func parseFibreChannelStatistics(hostPath string) (*FibreChannelCounters, error)
 	var counters FibreChannelCounters
 
 	path := filepath.Join(hostPath, "statistics")
+	if _, err := os.Stat(path); os.IsNotExist(err) {
+		return &counters, nil
+	}
 	files, err := os.ReadDir(path)
 	if err != nil {
```

Comment:
Why not check if the error is IsNotExist?

Reply:
I think it's better to just check if the statistics directory exists, instead of worrying about other errors like permission or filesystem issues. If the directory isn't there, the  function can keep going instead of stopping for fibrechannel statistics not found